residencialApp/user_management/views.py

```
@@ -0,0 +1,46 @@
+from django.shortcuts import render, redirect
+from django.contrib.auth import login, authenticate, logout
+from django.contrib.auth.forms import UserCreationForm
```

Ruff (0.8.2)

`django.contrib.auth.forms.UserCreationForm` imported but unused

Remove unused import: `django.contrib.auth.forms.UserCreationForm`

(F401)